Project/Assets/ML-Agents/Examples/FoodCollector/Scripts/FoodCollectorAgent.cs

```diff
@@ -30,6 +30,7 @@ public class FoodCollectorAgent : Agent
     public GameObject myLaser;
     public bool contribute;
     public bool useVectorObs;
+    public bool useFrozenFlag;
```

Comment:
I think this should always be turned on. The user might have issues with the number of observations when unchecking this box. Besides, you mentioned that the Agent will not train without it, so I think it should be always on and un-uncheckable.

Reply:
My concern was that it would be a bit confusing to use one vector flag by default in a "visual" scene. 
Or is it actually not too big a problem? if so I can make it un-uncheckable.